pom.xml

```diff
@@ -280,28 +238,19 @@
             <phase>validate</phase>
             <configuration>
               <rules>
-                <dependencyConvergence />
+                <dependencyConvergence>
+                  <!-- The items listed below have been verified non issue for allowing. -->
+                  <excludes>
+                    <exclude>org.eclipse.platform:org.eclipse.equinox.common</exclude>
+                    <exclude>org.eclipse.platform:org.eclipse.core.runtime</exclude>
+                    <exclude>org.osgi:org.osgi.util.function</exclude>
+                  </excludes>
+                </dependencyConvergence>
               </rules>
             </configuration>
           </execution>
         </executions>
       </plugin>
-      <plugin>
-        <groupId>org.apache.maven.plugins</groupId>
-        <artifactId>maven-dependency-plugin</artifactId>
-        <configuration>
-          <ignoredUnusedDeclaredDependencies>
-            <!-- ignore transitive dependency overridden to avoid vulnerability -->
-            <ignore>commons-beanutils:commons-beanutils</ignore>
-            <!-- ignore alignment with commons-logging to support slf4j -->
-            <ignore>commons-logging:commons-logging</ignore>
-            <!-- ignore unused osgi items added only for transitive dependency convergence-->
-            <ignore>com.google.errorprone:error_prone_annotations</ignore>
-            <ignore>org.osgi:org.osgi.util.function</ignore>
-            <ignore>org.osgi:osgi.annotation</ignore>
-          </ignoredUnusedDeclaredDependencies>
-        </configuration>
-      </plugin>
```

Comment:
note switching to dep management with this level of items did reduce the file by 51 lines.  So less to support.